rules/linux/process_creation/proc_creation_lnx_rev_shell_nc.yml

```diff
@@ -0,0 +1,29 @@
+title: Possible Nc or Ncat Reverse Shell
+id: 7f734ed0-4f47-46c0-837f-6ee62505abd9
+status: experimental
+description: Detects some of most common reverse shell techniques on Linux systems.
+references:
+      - https://pentestmonkey.net/cheat-sheet/shells/reverse-shell-cheat-sheet
+      - https://www.revshells.com/
+author: '@d4ns4n_'
+date: 2023/04/07
+tags:
+   - attack.initial_access
+   - attack.t1133
+logsource:
+   category: process_creation
+   product: linux
+detection:
+    selection_nc:
+        Image|contains: 'nc' #cover ncat
+        CommandLine|contains:
+            - '-r'
+            - '-e'
```

Comment:
Can't we add spaces here to avoid matches on longer strings in which an "-r" can appear? like `service-responder`